### DISCUSSION TITLE
[web] feature request: sqlglot / pyodide offline support

### DISCUSSION THREAD

**Discussion by barakplasma** [TobikoData/sqlmesh#3142] [Ideas]
can we inline pyodide / sqlglot into the web/client/src/assets folder for offline support? For environments with a firewall for example? Feels weird to make requests to jsdelivr.net every time I want to format some sql

**Comment by tobymao** [comment]
sure, would you like to make a PR? cc @mykalmax

**Reply by barakplasma** [reply]
yeah, thinking about making a PR. I opened #3148 to track it

**Reply by tobymao** [reply]
we can track it in the pr
